docs/_documentations/installoncloud.md

```diff
@@ -12,95 +12,47 @@ parent: root
+We recommend using the `chectl` CLI to install Eclipse Che for use with Codewind. For instructions on installing the `chectl` CLI tool, please see [Installing the chectl management tool](https://www.eclipse.org/che/docs/che-7/installing-the-chectl-management-tool/)
 
-**Note:** Complete the following steps before you create your `CheCluster` custom resource.
+Once `chectl` has been installed:
 
-1. Create the `eclipse-codewind` cluster role: `kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/master/setup/install_che/codewind-clusterrole.yaml`.
+1. Download the [codewind-che checluster yaml](https://github.com/eclipse/codewind-che-plugin/blob/master/setup/install_che/che-operator/codewind-checluster.yaml)  file to your machine
+    - Feel free to make modifications to this file as you wish, but we recommend leaving the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and `spec.storage.preCreateSubPaths` field set to `true`
+2. Determine your ingress domain. If you're unsure of your ingress domain, ask your cluster administrator
+2. Install Che:
+    - On OpenShift 3.x run the `chectl server:start --platform=openshift --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=<codewind-che.yaml file>`
+    - On regular Kubernetes run: `chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=<codewind-che.yaml file>`
 
-2. Ensure the following fields in your `CheCluster` `.yaml` file are set:
-- The `spec.server.cheWorkspaceClusterRole` field is set to `eclipse-codewind`, which was created when you previously applied the Codewind ClusterRole.
-- The `spec.storage.preCreateSubPaths` field is set to `true`.
-
-## Setting up OKD and OpenShift
-
-### Installing Che with deployment scripts
-
-If the Che operator was not used to install Eclipse Che, follow these steps instead. Otherwise skip these steps:
-
-1. Git clone the [Eclipse Che repository](https://github.com/eclipse/che).
-2. Enter the `cd` command to go to the `deploy/openshift` directory.
-3. Deploy Che with `./deploy_che.sh`.
-4. From the [Codewind Che plug-in repository](https://github.com/eclipse/codewind-che-plugin) run the `kubectl apply -f setup/install_che/codewind-clusterrole.yaml` command to create a cluster role with the required permission.
-5. From the [Codewind Che plug-in repository](https://github.com/eclipse/codewind-che-plugin) run the `kubectl apply -f setup/install_che/codewind-rolebinding.yaml` command.
-
-### Enable privileged and root containers to run
-Codewind is currently required to run as privileged (as it builds container images) and as root. Run the following commands to enable that functionality:
+## Enable privileged and root containers to run
+Codewind is currently required to run as privileged (as it builds container images) and as root. If your cluster is OpenShift 3.x, please run the following commands:
```

Comment:
- I'd remove "currently"
- Also remove the parenthesis and just say something like, "...run as root and as privileged because it builds container images."
- Remove "please"